src/components/Requirements/RequirementSideBar.vue

```diff
@@ -165,10 +184,10 @@ export default defineComponent({
   },
   props: {
     startTour: { type: Boolean, required: true },
+    isOpeningRequirements: { type: Boolean, required: true },
+    isDesktop: { type: Boolean, required: true },
   },
-  emits: {
-    showTourEndWindow: () => true,
-  },
+  emits: ['showTourEndWindow'],
```

Comment:
I think there's a reason why it was the way it was formatted in the previous way @SamChou19815 Can you provide insight?

Reply:
There was a second function after `showTourEndWindow` previously, which had a more complex function signature than `() => true`. That existing forced it to be in the more verbose format, but that function has been since removed. Not sure what makes the most sense stylistically but there are a bunch of other places in the code that do `emits: [...]`.